README.md

```diff
@@ -37,14 +37,14 @@ def info_dump(customers)
 end
 ```
 
-Snippet 3 - Big O:
+Snippet 3 - Big O: I am going to go with O(1) here because we are not comparing elements to each other and trying to figure out their order but comparing each to a constant so it should be the same amount of time for each.  
 ```ruby
 def first_element_is_red?(array)
   array[0] == 'red' ? true : false
 end
 ```
 
-Snippet 4 - Big O:
+Snippet 4 - Big O:  O(n*m) This one is stumping me... at first I thought two different arrays but its comparing items within an array to see if they are duplicate so O(n) seems a more like it, a linear time line that would increase the more data is added...   
```

Comment:
This is actually O(n^2) because we are iterating through `array` twice. Each time is O(n), and since these are nested, we get O(n^2)